src/test/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerModelTest.java

```diff
@@ -111,7 +108,7 @@ public void testDir() throws IOException {
 			ans.add(arr[0] + " " + arr[1] + " " + arr[2]);
 		}
 
-		modelSetUp(originalNetworkDir, EdgeWeightSetting.UNWEIGHTED, false); //create the model for algorithm
+		modelSetUp(originalNetworkDir, 23, EdgeWeightSetting.UNWEIGHTED, false); //create the model for algorithm with k = 23
```

Comment:
Add a description in the comments as to why you are testing these k values. Something like "enumerate all paths of length 2 to ensure the results will be the same. If we use a k that does not include all tied paths, results could be different due to the random nature of which paths PathLinker finds first"